src/openforms/submissions/api/views.py

```diff
@@ -96,3 +99,34 @@ def perform_create(self, serializer):
         verification = serializer.instance
         assert isinstance(verification, EmailVerification)
         transaction.on_commit(verification.send_email)
+
+
+@extend_schema(
+    summary=_("Verify email address"),
+    description=_(
+        "Using the code obtained from the verification email, mark the email address "
+        "as verified. Using an invalid code results in validation errors in the error "
+        "response."
+    ),
+)
+class VerifyEmailView(APIView):
+    authentication_classes = (AnonCSRFSessionAuthentication,)
+    permission_classes = (AnyActiveSubmissionPermission,)
+
+    def get_serializer_class(self):
```

Comment:
Why don't we add this as an attribute of the class and we override the method?I thought that we do that when we want to have more than one serializers depending on a specific situation.

Reply:
for drf-spectacular API documentation reasons :) otherwise it fails to properly process the serializer, because it's not a generic API view

Reply:
Nice, I wasn't aware of this!